runtime-modules/content/src/lib.rs

```diff
@@ -706,9 +676,8 @@ decl_module! {
             actor: ContentActor<T::CuratorGroupId, T::CuratorId, T::MemberId>,
             channel_id: T::ChannelId,
             params: ChannelUpdateParameters<T>,
+            assets: BTreeSet<DataObjectId<T>>,
```

Comment:
I suggest renaming it to `assets_to_remove` for more clarity. Perhaps also moving it to `ChannelUpdateParameters` may be a good idea? (and analogically with video)

Reply:
Ok I believe understand your thinking @Lezek123, ("Why do I need to specify the assets when I have to delete the whole bag regardless?") The problem lies in the fact that first the user is required to empty the channel bag by specifying objects ids. And there are not ways (at least that I am aware of) to retrieve such ids from the on-chain state